apps/demos/menuMeta.json

```diff
@@ -3313,6 +3313,24 @@
       }
     ]
   },
+  {
+    "Name": "Chat",
+    "Equivalents": "dxChat, dx-chat, chat",
```

Comment:
I guess we should also add`"Breadcrumb": true,`

Reply:
2679e0c3362d769e1ea3315578e64523e07689d8